packages/node_modules/node-red/settings.js

```diff
@@ -250,6 +250,14 @@ module.exports = {
      */
     //httpStaticRoot: '/static/',
 
+    /** The following property can be used to modify proxy options */
+    // proxyOptions: {
+    //     mode: "legacy", // legacy mode is for non-strict previous proxy determination logic (node-red < v4 compatible)
+    //     favourUpperCase: false, // Favour UPPER_CASE *_PROXY env vars
+    //     lowerCaseOnly: false, // Prevent UPPER_CASE *_PROXY env vars being used.
+    //     excludeNpm: false, // Prevent npm configured proxy env vars being used.
```

Comment:
```suggestion
```

Remove this from visible options (hide API, i.e. make it subject to change/removal)